tests/contrib/langchain/test_langchain_llmobs.py

```diff
@@ -601,6 +733,13 @@ def _call_openai_llm(OpenAI):
         with get_request_vcr(subdirectory_name="langchain_community").use_cassette("openai_completion_sync.yaml"):
             llm.invoke("Can you explain what Descartes meant by 'I think, therefore I am'?")
 
+    @staticmethod
+    def _call_openai_embedding(OpenAIEmbeddings):
+        embedding = OpenAIEmbeddings()
+        with mock.patch("langchain_openai.OpenAIEmbeddings._get_len_safe_embeddings", return_value=[0.0] * 1536):
```

Comment:
```suggestion
```
When running with OpenAI, we do not want to mock out `_get_len_safe_embeddings()` because that involves the call to `OpenAI.embeddings.create(...)`.